MessagePack.xcodeproj/xcshareddata/xcschemes/MessagePack watchOS.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "825321D21EFFCAED00914B55"
-               BuildableName = "MessagePack-watchOS.framework"
+               BuildableName = "Messagepack.framework"
```

Comment:
Should this be `MessagePack` with a capital P?

Reply:
Fixed.